packages/zosfiles/src/methods/create/Create.ts

```diff
@@ -134,7 +134,10 @@ export class Create {
         }
     }
 
-    public static async dataSetLike(session: AbstractSession, dataSetName: string, likeDataSetName: string): Promise<IZosFilesResponse> {
+    public static async dataSetLike(session: AbstractSession,
+                                    dataSetName: string,
+                                    likeDataSetName: string,
+                                    options?: Partial<ICreateDataSetOptions>): Promise<IZosFilesResponse> {
```

Comment:
I like this approach to add optional argument so there isn't a breaking change 👍